Source/TwilightPrincess/rules.txt

```diff
@@ -0,0 +1,217 @@
+[Definition]
+titleIds = 000500001019C800,000500001019E600,000500001019E500
+name = Resolution
+path = "The Legend of Zelda: Twilight Princess HD/Graphics/Resolution"
+description = Changes the resolution of the game.
+version = 3
+
+[Preset]
+name = 1280x720 Default
+$width = 1280
+$height = 720
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 256x144
+$width = 256
+$height = 144
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 427x240
+$width = 427
+$height = 240
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 640x360
+$width = 640
+$height = 360
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 854x480
+$width = 854
+$height = 480
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 960x540
+$width = 960
+$height = 540
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 1600x900
+$width = 1600
+$height = 900
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 1920x1080
+$width = 1920
+$height = 1080
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 2560x1440
+$width = 2560
+$height = 1440
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 2732x1536
+$width = 2732
+$height = 1536
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 3200x1800
+$width = 3200
+$height = 1800
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 3840x2160 (4k - Native x2)
+$width = 3840
+$height = 2160
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 5120x2880
+$width = 5120
+$height = 2880
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 7680x4320
+$width = 7680
+$height = 4320
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 10240x5760
+$width = 10240
+$height = 5760
+$gameWidth = 1280
+$gameHeight = 720
+
+
+[TextureRedefine]
+width = 1920
+height = 1088
+#formats = 0x011,0x01a
+overwriteWidth = ($width/$gameWidth) * 1920
+overwriteHeight = ($height/$gameHeight) * 1088
+
+[TextureRedefine]
+width = 1920
+height = 1080
+#formats = 
+overwriteWidth = ($width/$gameWidth) * 1920
+overwriteHeight = ($height/$gameHeight) * 1080
+
+[TextureRedefine]
+width = 960
+height = 544
+#formats = 0x01a,0x011
+overwriteWidth = ($width/$gameWidth) * 960
+overwriteHeight = ($height/$gameHeight) * 544
+
+[TextureRedefine]
+width = 960
+height = 540
+#formats = 0x01a
+overwriteWidth = ($width/$gameWidth) * 960
+overwriteHeight = ($height/$gameHeight) * 540
+
+[TextureRedefine]
+width = 864
+height = 480
+#formats = 
+overwriteWidth = ($width/$gameWidth) * 864
+overwriteHeight = ($height/$gameHeight) * 480
+
+[TextureRedefine]
+width = 854
+height = 480
+#formats = 
+overwriteWidth = ($width/$gameWidth) * 854
+overwriteHeight = ($height/$gameHeight) * 480
+
+[TextureRedefine] #shadows horsie
+width = 768
+height = 768
+formatsExcluded = 
+overwriteWidth = ($width/$gameWidth) * 768
+overwriteHeight = ($height/$gameHeight) * 768
```

Comment:
shouldn't shadows go to their own packs?

Reply:
Sure as an option, but why remove them from the res pack? 
By default: Scale all viewports, shadows, pad,  LOD etc..  ie a complete package that scales in "quality"  with resolution

If you then want to change shadows res, have 4k gamepad, gaussian dof, colour changes etc  we offer separate packs as well.

But It's your project, just decide either way.

Reply:
*bump*